common/views/components/FindUs/FindUs.tsx

```diff
@@ -35,7 +34,7 @@ const StyledFindUs = styled.div.attrs(props => ({
   }
 `;
 
-const FindUs = () => (
+const FindUs: FunctionComponent = (): ReactElement => (
```

Comment:
We don't need the `ReactElement` return type... clutters the file in my 👀

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/v16/index.d.ts#L547

You can see the return type is here ☝️

Reply:
nice! ill do that